lms/djangoapps/course_blocks/tests/test_utils.py

```diff
@@ -0,0 +1,32 @@
+"""
+Helpers for Course Blocks tests.
+"""
+
+from openedx.core.lib.block_structure.cache import BlockStructureCache
+from openedx.core.lib.block_structure.transformer_registry import TransformerRegistry
+from ..api import _get_cache
+
+
+class EnableTransformerRegistryMixin(object):
+    """
+    Mixin that enables the TransformerRegistry to USE_PLUGIN_MANAGER for
+    finding registered transformers.  USE_PLUGIN_MANAGER is set to False
+    for LMS unit tests to speed up performance of the unit tests, so all
+    registered transformers in the platform do not need to be collected.
+    This Mixin is expected to be used by Tests for integration testing
+    with all registered transformers.
+    """
+    def setUp(self, **kwargs):
+        super(EnableTransformerRegistryMixin, self).setUp(**kwargs)
+        TransformerRegistry.USE_PLUGIN_MANAGER = True
```

Comment:
You mean, why a Mixin?  Mostly for reducing code duplication across all affected tests and encapsulating the mocking behavior.